src/utils/isAfterDay.js

```diff
@@ -0,0 +1,18 @@
+import moment from 'moment';
+
+export default function isAfterDay(a, b) {
```

Comment:
I don't mean, let's get rid of both - I mean, let's implement one in terms of the other.